<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPlaybackControllerGuiComp</class>
 <widget class="QWidget" name="CPlaybackControllerGuiComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="PlayButton">
     <property name="text">
      <string>Play</string>
     </property>
     <property name="icon">
      <iconset resource="../iqtgui/iqtgui.qrc">
       <normaloff>:/Icons/Play.svg</normaloff>:/Icons/Play.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="TimeEdit">
     <property name="maximumTime">
      <time>
       <hour>0</hour>
       <minute>59</minute>
       <second>59</second>
      </time>
     </property>
     <property name="displayFormat">
      <string>H:mm:ss.zzz</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="PositionSlider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="IndexFrame" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="FrameIndexSB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MaxFramesLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="RepeatButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../iqtgui/iqtgui.qrc">
       <normaloff>:/Icons/Reload.svg</normaloff>:/Icons/Reload.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../iqtgui/iqtgui.qrc"/>
 </resources>
 <connections/>
</ui>
